espoon-voltti/evaka
# EVAKA-4045: Adjust service voucher decision texts

#### Summary

- Widen create new person modal
- Reorganise PDF creation and testing, add voucher generation test
- Do not use DaycareFormV0 when generating PDFs
- Refactor and simplify DecisionService usage
- Update private service voucher decision texts


#### How to test

- Run tests in `PDFServiceTest`. Tests will print paths to generated PDFs in the logs. Check both, Finnish and Swedish versions
- Uncomment `//File("/tmp/fee_decision_test.pdf").writeBytes(pdfBytes)` in `PdfServiceTest` and run `generateFeeDecisionPdf smoke test`. (Yes, I know, naming sucks). Check the generated fee decision in `/tmp/fee_decision_test.pdf`. Change the language in the test to generate Swedish version of fee decision.
